src/lib/ng-package-format/package.ts

```diff
@@ -66,6 +66,13 @@ export class NgPackage {
     return this.absolutePathFromPrimary('workingDirectory');
   }
 
+  public get whitelistedNonPeerDependencies(): string[] {
+    // XX: default array values from JSON schema not recognized
+    const defValue = ['tslib'];
+
+    return this.primary.$get('whitelistedNonPeerDependencies') || defValue;
```

Comment:
This way, if you specify `whitelistedNonPeerDependencies` yourself, you will also have to include `tslib` yourself. Wouldn't it be easier to just always include `tslib` instead?

Reply:
Good question. Is `tslib` necessarily required? Other way round: is it possible to build a package that does not depend on `tslib`?

Reply:
The case would be: "I, as library author, wrote a package that does not depend on `tslib` and I'd like to verify that I don't accidentally install it via `dependencies`"

But we always add `tslib` regardless of it's being required or not. So we'd need to check the ESM bundles for an import (via AST to be reliable) to determine that.

Reply:
@dherges hmm, not sure, either choice is somewhat arbitrary. In the spirit of keeping it simple I wouldn't go to great lengths in establishing whether tslib is actually used or not.